crates/loam-soroban-sdk/README.md

```diff
@@ -1 +1,3 @@
-# loam-sdk
+# loam-soroban-sdk
+
+Loam wrapper around Soroban SDK. This version of Soroban SDK would allow the use of Loam.
```

Comment:
```suggestion
Wrapper around Soroban SDK which adds features needed by Loam SDK, such as:

 - lazy getting and setting
 - implementation of Loam's`IntoKey` derive macro that relies on Soroban SDK's specifics
 - abstraction of Soroban SDK's `env` to enable Loam SDK-authored contracts to avoid referencing it at all and use Rust's standard mutable/immutable method definition syntax (`my_method(&self, ...)` vs `my_method(&mut self, ...)`).
```